packages/ethereum-ledger-provider/lib/EthereumLedgerProvider.js

```diff
@@ -57,11 +61,11 @@ export default class EthereumLedgerProvider extends LedgerProvider {
     const path = address.derivationPath
 
     let txData = {
-      to: to ? ensureHexEthFormat(to) : undefined,
+      to: to ? ensureHexEthFormat(to) : null,
       from: ensureHexEthFormat(from),
-      value: BigNumber(value).toNumber(),
+      value: ensureHexEthFormat(BigNumber(value).toString(16)),
```

Comment:
Similarly with these changes. Thought that they were already in dev.

Reply:
i couldn't cherry pick jules' commits so i had to copy a few changes. only four lines were there.